build.gradle.kts

```diff
@@ -42,3 +51,35 @@ tasks.named<Test>("test") {
     // Use JUnit Platform for unit tests.
     useJUnitPlatform()
 }
+
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// Static Analysis Tasks
+
+spotless {
+    java {
+        // note: you can use an empty string for all the imports you didn't specify explicitly, '|' to join group without blank line, and '\\#` prefix for static imports
+        importOrder("java|javax", "org.hibernate", "com.mongo", "", "\\#")
```

Comment:
Just curious why you want a non-default import order. Is the default not good?

Reply:
the main reason is to add group blank line delimiter for the two main importing groups. Not a big deal for sure.

Reply:
I reomved both for I think it is over engineering for now